connexion/apps/aiohttp_app.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, import_name, only_one_api=False, **kwargs):
         self._api_added = False
 
     def create_app(self):
-        return web.Application(debug=self.debug)
```

Comment:
hey @Jyhess , this doesn't seem related to json serialization.
Can you factor this out into a separate PR.

Reply:
I will do it